Guide/installation.markdown

```diff
@@ -116,6 +116,15 @@ Note that nix can gradually grow to use several GB of disk space, especially aft
 
 You can now install IHP by running:
 
+```bash
+nix-env --install ihp-new
+```
+
```

Comment:
How about mentioning `ihp-new` can also be added to the NixOS or Home Manager list of packages?

E.g.:
```
If you use Home Manager or NixOS, you can also add `ihp-new` to the list of packages.
```

Reply:
Related but different: as an alternative, users may try without installing by running either `nix-shell --packages ihp-new` or `nix shell nixpkgs#qmmp`.

How about also documenting it? Or is it overkill?

The reason I ask is because Nix/NixOS encourage declarative style/reproducibility, and installing with `nix-env -i` kinda goes against :sweat_smile:

Reply:
> How about mentioning ihp-new can also be added to the NixOS or Home Manager list of packages?

👍  

> How about also documenting it? Or is it overkill?

I think these are all things only more experienced nix people know about. The documentation here is mostly not aimed at those people, the experts already know how to do it their own way and will skip our instructions. Also `nix shell` currently doesn't work without explicitly enabling it in the nix config (https://github.com/NixOS/nix/issues/7000).

Reply:
Fair enough!